### ISSUE TITLE
\\ sign in hostname breaks DB Connection #633

### ISSUE THREAD

**Issue by mehmetkara5007** [denisenkom/go-mssqldb#634]
I am using different signs for variables of password, username and host name.
I can wrap chars  @,#,$,%,& by using Url.QueryEscape()

But I can not deal with "\\" and "|" inside host name. (myapp.abc\\snd or myapp.abc|snd)
I am taking error: **invalid URL escape \"%7C\""**  even if I use url.PathEscape.

How can I solve this problem?

Thanks

**Comment by mehmetkara5007** [comment]
any help?

**Comment by denysvitali** [comment]
I don't think that an hostname can have a backslash. You are probably looking at `hostname/instance`

**Comment by mehmetkara5007** [comment]
> have a backslash. You are pro

I have 2 back slash for hostname in configuration page. It was working good with GORM 1

**Comment by tc-hib** [comment]
Hello,

Instances use one backslash: `hostname\instance`

If you read the [Readme](https://github.com/denisenkom/go-mssqldb#the-connection-string-can-be-specified-in-one-of-three-formats), you'll see you should replace the backslash by a slash in the URL.
Or you can use another kind of connection string : ADO or ODBC.
